src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const allRows = [
+  ...table.querySelectorAll('thead tr'),
+  ...table.querySelectorAll('tbody tr'),
+  ...table.querySelectorAll('tfoot tr'),
+];
+
+allRows.forEach((row) => {
+  const secondCell = row.cells[1];
+
+  const clonedCell = secondCell.cloneNode(true);
+
+  row.insertBefore(clonedCell, row.cells[row.cells.length - 1]);
+});
```

Comment:
Consider adding a check to ensure that each row has at least two cells before accessing `row.cells[1]`. This will prevent potential runtime errors if a row has fewer than two cells.